lib/set_builder/modifier/base.rb

```diff
@@ -79,12 +81,20 @@ def errors_with_number_value(string)
 
 
       def to_s(negative=false)
-        words = negative ? [self.class.negate(operator).to_s.gsub(/_/, " ")] : [operator.to_s.gsub(/_/, " ")]
-        arguments = self.class.operators[operator] || []
-        (0...arguments.length).each do |i|
-          words << ValueMap.to_s(arguments[i], values[i])
+        arguments = self.class.parsed_operators[operator] || Arguments.new("")
```

Comment:
:arrow_down: You've got this line here and on L48. Would it make sense to extract to a method?